components/pedigree/resources/src/main/resources/pedigree/view/datepicker.js

```diff
@@ -248,7 +250,7 @@ define([
               values.push({'value': v, 'text' : ("0" + v).slice(-2)});
           }
       } else {
-          for (var v = end; v <= start; --v) {
+          for (var v = start; v >= end; --v) {
```

Comment:
If the loop should really start at `end`, then it needs to be: `for (var v = end; v <= start; ++v) {`

Reply:
change is correct